project/client/static/scripts/script.js

```diff
@@ -21,97 +21,98 @@ const toInt = (str) => {
     }
 };
 
+/**
+ * Shorthand for document.getElementById()
+ * @param str
+ * @returns {object}
+ */
+function $(id) {
```

Comment:
This is okay for me, but only because jQuery is so popular. Normally I would much rather see expressive naming.